--- freexl-2.0.0.orig/makefile.vc	2023-09-15 20:57:15.072357562 -0500
+++ freexl-2.0.0/makefile.vc	2023-09-15 20:58:49.590348078 -0500
@@ -11,7 +11,7 @@
 LIBOBJ	               =	$(CORE) $(XLSX) $(ODS)
 FREEXL_DLL	 	       =	freexl$(VERSION).dll
 
-CFLAGS	=	/nologo -I. -Iheaders -IC:\OSGeo4W\include $(OPTFLAGS)
+CFLAGS	=	/nologo -I$(LIBRARY_INC) -I. -Iheaders $(OPTFLAGS)
 
 default:	all
 
@@ -35,10 +35,9 @@
 freexl_i.lib:	$(LIBOBJ)
 	link /debug /dll /out:$(FREEXL_DLL) \
 		/implib:freexl_i.lib $(LIBOBJ) \
-		C:\OSGeo4W\lib\iconv.lib \
-		C:\OSGeo4W\lib\libexpat.lib \
-		C:\OSGeo4W\lib\libminizip.lib \
-		C:\OSGeo4w\lib\zlib.lib
+		$(LIBRARY_LIB)\iconv.lib \
+		$(LIBRARY_LIB)\libexpat.lib \
+		$(LIBRARY_LIB)\libminizip.lib
 	if exist $(FREEXL_DLL).manifest mt -manifest \
 		$(FREEXL_DLL).manifest -outputresource:$(FREEXL_DLL);2 
 
