--- proj-9.2.0.orig/src/tests/geodsigntest.c	2023-03-01 04:58:50.000000000 -0300
+++ proj-9.2.0/src/tests/geodsigntest.c	2023-03-02 13:59:56.543419844 -0300
@@ -40,7 +40,7 @@
  * serious (just a loss of accuracy).  If you're still using the buggy glibc,
  * then define OLD_BUGGY_REMQUO to be 1.
  */
-#define OLD_BUGGY_REMQUO 0
+#define OLD_BUGGY_REMQUO 1
 #endif
 
 static const T wgs84_a = 6378137, wgs84_f = 1/298.257223563; /* WGS84 */
