Index: work/CMakeLists.txt
===================================================================
--- work.orig/CMakeLists.txt
+++ work/CMakeLists.txt
@@ -176,9 +176,9 @@ endif()
 
 # Would build and run with older versions, but with horrible performance
 # See https://github.com/OSGeo/PROJ/issues/1718
-if("${SQLITE3_VERSION}" VERSION_LESS "3.11")
-  message(SEND_ERROR "sqlite3 >= 3.11 required!")
-endif()
+#if("${SQLITE3_VERSION}" VERSION_LESS "3.11")
+#  message(SEND_ERROR "sqlite3 >= 3.11 required!")
+#endif()
 
 ################################################################################
 # Check for libtiff
